src/ts/localization/i18n.ts

```diff
@@ -41,6 +41,12 @@ interface Vocabulary {
   'settings.subtitles.font.family.smallcapital': string;
   'settings.subtitles.window.color': string;
   'settings.subtitles.window.opacity': string;
+  'settings.ariaLabel.volume': string;
+  'settings.ariaLabel.mute': string;
+  'settings.ariaLabel.play': string;
```

Comment:
Please make them generic labels. Constraining them to "aria" labels unnecessarily reduces their versatility.

Reply:
For play I can do it but for volume not that easy since we have defined `"settings.audio.mute" : "Volume/Mute"` Which should actually be `settings.audio.volumeMute` and I can't define both of them separately

Reply:
Then make the text labels on those buttons stateful as well?

Reply:
We cannot make it stateful as it doesn't make sense for the client:

https://github.com/bitmovin/player-issues-web/issues/2586#issuecomment-560999083

Feedback number 4.

Reply:
The answer you lined is exactly demanding an aria "state", which is what I meant with "stateful". if that is implemented for accessibility, why not do it for the "normal" label as well?

Reply:
Because we already have it open for other clients and changing the "normal" labels can be api breaking 😞

Reply:
The ID is called `settings.audio.mute` so please change the string from `Volume/Mute` to `Mute`, and then add a `settings.audio.volume` with string `Volume`. Also add a `settings.audio.unmute` => `Unmute` label which is the label that should be set while the player is muted.